examples/generalized_linear_models/GLM-discrete-choice_models.ipynb

```diff
@@ -0,0 +1,5710 @@
+{
```

Comment:
<p>You can use <code>pm.math.softmax</code> to avoid that annoying warning </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Thanks, yeah I'll adjust that. I think the modelling strategy is coming along now. I got overly hung up on the long-data format stan was using. Sticking with the wide data format for the minute. Seems more transparent to me and better pedagogically.